recipes/qwt/all/conanfile.py

```diff
@@ -75,6 +74,8 @@ def validate(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.options.designer:
+            self.options["qt"].qttools = True
```

Comment:
Conan-center never builds qt with qttools enabled, so this will fail with a missing binary for qt

Reply:
Without that, leaving "designer" option at true by default as in 6.1.6 release, give me following message:

```
CMake Error at designer/CMakeLists.txt:21 (target_link_libraries):
  Target "qwt_designer_plugin" links to:

    Qt5::Designer

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
  
```

Then what to do?
Leave "designer" option at false by default in 6.2.0 or hard code that feature at OFF? 

```
tc.variables["QWT_DESIGNER"] = "OFF"
```

Regards.

Reply:
leave the default designer at False is the way to go